modules/external_commands/inc/com/centreon/engine/modules/external_commands/processing.hh

```diff
@@ -217,10 +216,40 @@ namespace         modules {
 
         char* name(my_strtok(args, ";"));
         char* description(my_strtok(NULL, ";"));
-        service* svc(::find_service(name, description));
-        if (!svc)
+
+        std::pair<uint64_t, uint64_t> found_id(get_host_and_service_id(
+          name, description));
+        std::unordered_map<std::pair<uint64_t, uint64_t>,
+                           std::shared_ptr<service> >::const_iterator
+          found(configuration::applier::state::instance().services().find(found_id));
+
+        if (found != configuration::applier::state::instance().services().end() ||
+          found->second.get() != nullptr)
```

Comment:
Isn't it wrong?
`if (!svc)` becomes `found->second.get() != nullptr` ...
I think you wanted` !found->second`

Reply:
done